src/test/kotlin/no/nav/familie/ef/iverksett/ServerTest.kt

```diff
@@ -0,0 +1,55 @@
+package no.nav.familie.ef.iverksett
+
+import com.github.tomakehurst.wiremock.WireMockServer
+import no.nav.familie.ef.iverksett.infrastruktur.database.DbContainerInitializer
+import no.nav.familie.ef.iverksett.util.onBehalfOfToken
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.extension.ExtendWith
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.boot.test.web.client.TestRestTemplate
+import org.springframework.boot.web.server.LocalServerPort
+import org.springframework.context.ApplicationContext
+import org.springframework.http.HttpHeaders
+import org.springframework.test.context.ActiveProfiles
+import org.springframework.test.context.ContextConfiguration
+import org.springframework.test.context.junit.jupiter.SpringExtension
+
+@ExtendWith(SpringExtension::class)
+@ContextConfiguration(initializers = [DbContainerInitializer::class])
+@SpringBootTest(classes = [ApplicationLocal::class], webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@ActiveProfiles("servertest", "mock-oppdrag")
+abstract class ServerTest {
+
+    protected val restTemplate = TestRestTemplate()
+    protected val headers = HttpHeaders()
+
+    @Autowired
+    private lateinit var applicationContext: ApplicationContext
+
+    @LocalServerPort
+    private var port: Int? = 0
+
+    @AfterEach
+    fun reset() {
+        resetWiremockServers()
```

Comment:
@throndi @olekvernberg @neringaalt 
PS: Inni `reset` vil det være naturlig å slette alle rader fra alle tabeller mellom hver test-kjøring for å ha en "clean state" for hver test.